internal/translation/networkcontainer/conversion.go

```diff
@@ -73,3 +99,32 @@ func fromAtlasNoStatus(container *admin.CloudProviderContainer) *NetworkContaine
 		ID: container.GetId(),
 	}
 }
+
+func fromAtlasAWSStatus(container *admin.CloudProviderContainer) *AWSContainerStatus {
+	if container.VpcId == nil {
+		return nil
+	}
+	return &AWSContainerStatus{
+		VpcID: container.GetVpcId(),
+	}
```

Comment:
Just a hint on how SDK expect such cases to be handled.
```suggestion
	if ok, vpcID := container.GetVpcIdOk(); ok {
		return &AWSContainerStatus{
		    VpcID: vpcID,
	        }
	}
	return nil
```